src/app/services/user.service.ts

```diff
@@ -0,0 +1,24 @@
+import { Injectable } from '@angular/core';
+import {HttpClient, HttpResponse} from '@angular/common/http';
+import { UserRegister } from '../models/register.model';
+import { Observable } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class UserService {
+  /**
+   * This is base url link which is common for all
+   *  */
+  private baseUrl = 'https://localhost:44329';
+
+  constructor(private http : HttpClient) { }
+
+   /**
+   * addUsers are used to post data in the Api
+   * @retun http response
```

Comment:
Make sure to add description of params & returns not only the return type and the param type.
https://marketplace.visualstudio.com/items?itemName=salbert.comment-ts.
Take care of this from next PR without fail

Reply:
OK Dii I will take care of this in the next PR.